src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1155,6 +1159,9 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
                                                           ? bundle.getPossibleRecipients(question, participantIdentifier)
                                                           : bundle.getPossibleGivers(question, participantIdentifier);
 
+        int responseRecipientIndex = 0;
```

Comment:
There is no need for the declaration of `responseRecipientIndex` and `responseGiverIndex` here right?